build-scripts/bundle.cjs

```diff
@@ -90,6 +90,8 @@ module.exports.babelOptions = ({ latestBuild, isProdBuild, isTestBuild }) => ({
     setSpreadProperties: true,
   },
   browserslistEnv: latestBuild ? "modern" : "legacy",
+  // Must be unambiguous because some dependencies are CommonJS only
+  sourceType: "unambiguous",
```

Comment:
Are we sure all our files have `imports` or `exports`? Apart from the ones you added it too in this PR?

Reply:
Can we add some kind of test or lint rule for this?

Reply:
For the `*.ts` files, TS will throw an error now with `isolatedModules: true`.  That's how I found them.

For the `*.js` files, I checked manually, but if we turn on the TS flag to check JS files it should error there too.  I didn't do that here because I expect it would be another PR's worth of type fixes.